examples/PFHub1a.cpp

```diff
@@ -18,15 +18,15 @@ int main(int argc, char* argv[]) {
             std::string problem_name(argv[1]);
             if (problem_name == "benchmark")
                 simulation = std::make_unique<PFHub1aBenchmark>(grid_points, dt);
-            else if (problem_name == "periodic")
-                simulation = std::make_unique<PFHub1aPeriodic>(grid_points, dt);
+            else if (problem_name == "2023")
+                simulation = std::make_unique<PFHub1aCHiMaD2023>(grid_points, dt);
             else
                 throw std::invalid_argument("");
 
             simulation->run_until_time(end_time);
             simulation->output_c();
```

Comment:
Right now you assume we only want free energy for the new comparison - shouldn't both of these outputs be optional? 
```
if (output_composition)
    simulation->output_c();
if (output_free_energy)
    simulation->free_energy();
```

Reply:
This will be included in my executable-rework PR